playbooks/repec.yml

```diff
@@ -1,9 +1,12 @@
 ---
-- hosts: repec_prod
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- hosts: repec_{{ runtime_env | default('staging') }}
   remote_user: pulsys
   become: true
   vars_files:
-    - ../group_vars/repec/repec_prod.yml
+    - ../group_vars/repec/repec_{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/repec/common.yml
     - ../group_vars/repec/vault.yml
```

Comment:
Should this playbook include the datadog role for production? Is it a conscious choice to leave it off?

Reply:
Good idea!  I added the datadog role.